gulpfile.js

```diff
@@ -10,6 +10,8 @@ const git = require('gulp-git-streamed');
 const gulp = require('gulp');
 const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
+const lockfileVersions = [1];
```

Comment:
How about `supportedLockFileVersion`, and make it a single value rather than a list. If we get to the point that we need to support a new version, we'll likely need to re-evaluate this code anyway.